app/Invoices/Subscription/Bill.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Invoices\Subscription;
+
+use DateTime;

```

Comment:
netiek izmantots imports. ja nav nepieciešams - dzēšam laukā. YAGNI (You Aint Gonna Need It) metode